src/code/PublishPSResource.cs

```diff
@@ -167,8 +167,8 @@ protected override void BeginProcessing()
                 ThrowTerminatingError(
                     new ErrorRecord(
                         new ArgumentException(
-                            "The path to the resource to publish is not in the correct format, point to a path or file of the module or script to publish."),
-                            "InvalidSourcePath",
+                            "The path to the resource to publish is not in the correct format or does not exist. Please provide the path of the root module or the path to the .psd1."),

```

Comment:
would it maybe be clearer what we mean by root module if we provide an example in the instruction?
```suggestion
                            "The path to the resource to publish is not in the correct format or does not exist. Please provide the path of the root module (i.e `./ModuleToPublish/`) or the path to the .psd1 (i.e `./ModuleToPublish/ModuleToPublish.psd1`"),

```